Documentation/ceph-nfs-crd.md

```diff
@@ -66,6 +70,12 @@ spec:
     priorityClassName:
 ```
 
+ You can enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool with the following command in the ceph-tools container:
+ 
+```yaml 
+ceph dashboard set-ganesha-clusters-rados-pool-namespace <your cluster name>:<the ganesha pool name>/<the ganesha namespace> 
```

Comment:
```suggestion
$ ceph dashboard set-ganesha-clusters-rados-pool-namespace <pool_name>[/<namespace>]
```
see [this](https://docs.ceph.com/en/latest/mgr/dashboard/#configuring-nfs-ganesha-in-the-dashboard)

Reply:
Yaa it's the way given in the document but I would say that's the fix I have to do for the command, [quoted here ](https://github.com/rook/rook/issues/7064#issuecomment-790819657)

Reply:
Yaa the above command is also true for single cluster but for using multiple NFS-Ganesha clusters we require [cluster name or id](https://docs.ceph.com/en/latest/mgr/dashboard/#support-for-multiple-nfs-ganesha-clusters).

Reply:
in that case, you can have 2 commands one for a single cluster and the other for multiple clusters.

Reply:
An example of cluster name, ganesha pool name, and ganesha namespace that match examples we have would be good for users also.

Reply:
ok